interp/src/stk_env.rs

```diff
@@ -772,6 +773,7 @@ impl<K: Eq + std::hash::Hash, V: Eq> Smoosher<K, V> {
     /// ```
     /// will panic:
     /// ```should_panic
```

Comment:
Are the example tests actually run anywhere in the CI?

Reply:
No, they are not. Which is why I had to fix a bunch of them when I finally thought to check.